docs/extensions/controllerregistration.md

```diff
@@ -105,7 +105,7 @@ gardener:
   garden:
     identifier: <uuid-of-gardener-installation>
   seed:
-    identifier: <seed-name>
+    name: <seed-name>
```

Comment:
WDYT about documenting all values?
```yaml
gardener:
  version: <gardener-version>
  garden:
    clusterIdentity: <uuid-of-gardener-installation>
    genericKubeconfigSecretName: <generic-garden-kubeconfig-secret-name>
  seed:
    name:             <seed-name>
    clusterIdentity:  <seed-cluster-identity>
    annotations:      <seed-annotations>
    labels:           <seed-labels>
    provider:         <seed-provider-type>
    region:           <seed-region>
    volumeProvider:   <seed-first-volume-provider>
    volumeProviders:  <seed-volume-providers>
    ingressDomain:    <seed-ingress-domain>
    protected:        <seed-protected-taint>
    visible:          <seed-visible-setting>
    taints:           <seed-taints>
    networks:         <seed-networks> 
    blockCIDRs:       <seed-networks-blockCIDRs>
    spec:             <seed-spec>
  gardenlet:
    featureGates: <gardenlet-feature-gates>
```

Reply:
Sure, If the component owners are also OK and confirm
 that those fields are indeed required. Since those have not been documented until now, most likely those are not used. However whether this was the original intent or not, only the authors can advise on.

Reply:
These are all the values that are passed. I would rather assume that new values were added over time but we didn't document them.
The question whether all of the values are used somewhere by some extension or whether the list can be optimized (for example now we pass the seed spec and bunch of other things that can be fetched from the seed spec) - I think this deserve a dedicated issue for follow-up.
I would vote to document all values. Any reason that we should not be doing this?

Reply:
I pushed my suggestion as a new commit (https://github.com/gardener/gardener/pull/9141/commits/41c12b1af9044a8e6dad9f6e71d4ab7431fb1e7f).